cmd/operator-ui/vendor/github.com/blackducksoftware/synopsys-operator/pkg/api/blackduck/v1/types.go

```diff
@@ -39,18 +39,20 @@ type Blackduck struct {
 
 // BlackduckView will be used to populate information for the Blackduck UI.
 type BlackduckView struct {
-	Clones           map[string]string `json:"clones"`
-	StorageClasses   map[string]string `json:"storageClasses"`
-	CertificateNames []string          `json:"certificateNames"`
-	Environs         []string          `json:"environs"`
-	ContainerTags    []string          `json:"containerTags"`
-	Version          string            `json:"version"`
+	Clones            map[string]string `json:"clones"`
+	StorageClasses    map[string]string `json:"storageClasses"`
+	CertificateNames  []string          `json:"certificateNames"`
+	Environs          []string          `json:"environs"`
+	ContainerTags     []string          `json:"containerTags"`
+	Version           string            `json:"version"`
+	SupportedVersions []string          `json:"supportedVersions"`
 }
 
 // BlackduckSpec will be CRD Blackduck definition's Spec
 type BlackduckSpec struct {
 	Namespace         string                    `json:"namespace"`
 	Size              string                    `json:"size"`
+	Version           string                    `json:"version"`
```

Comment:
Instead of creating a new version, whether it is fine to add in a same version. I don't know on how many customers using the operator